Improve Player Lifecycle Docs (#869)
spatialos/gdk-for-unity: docs/modules/player-lifecycle/overview.md

<%(TOC)%>
# Player Lifecycle Feature Module

> The Player Lifecycle Feature Module provides you with an implementation of player creation and simple player lifecycle management. You can find the Feature Module in the GDK repository [here](https://github.com/spatialos/gdk-for-unity/tree/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle).

Reviewer: ```suggestion
The Player Lifecycle Feature Module provides player creation functionality and simple player lifecycle management. You can find the Feature Module in the GDK repository [here](https://github.com/spatialos/gdk-for-unity/tree/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle).
```

The module includes the following features:

* Send player creation requests and handle responses.
* Ability to provide arbitrary serialized data as part of the player creation loop.

> * Send and acknowledge player heartbeats. [What's a heartbeat?]({{urlRoot}}/modules/player-lifecycle/heartbeating)

Reviewer: ```suggestion
* Send and acknowledge player [heartbeats]({{urlRoot}}/modules/player-lifecycle/heartbeating).
```
